source/reference/minio-cli/minio-mc-admin/mc-admin-speedtest.rst

```diff
@@ -0,0 +1,64 @@
+======================
+``mc admin speedtest``
+======================
+
+.. default-domain:: minio
+
+.. contents:: Table of Contents
+   :local:
+   :depth: 2
+
+.. mc:: mc admin speedtest
+
+Description
+-----------
+
+.. start-mc-admin-speedtest-desc
+
+The :mc-cmd:`mc admin speedtest` command tests throughputs per host with PUT and GET operations.
+
+:mc-cmd:`mc admin speedtest` is not supported on standalone MinIO deployments
+or MinIO Gateway deployments.
+
+.. end-mc-admin-speedtest-desc
+
+.. admonition:: Use ``mc admin`` on MinIO Deployments Only
+   :class: note
+
+   .. include:: /includes/facts-mc-admin.rst
+      :start-after: start-minio-only
+      :end-before: end-minio-only
+
+Syntax
+------
+
+:mc-cmd:`mc admin speedtest` has the following syntax:
+
+.. code-block:: shell
+   :class: copyable
+
+   mc admin speedtest [FLAGS] TARGET
+
+:mc-cmd:`mc admin speedtest` supports the following arguments:
+
+.. mc-cmd:: TARGET
+
+   *Required*
+
+   The :mc-cmd:`alias <mc alias>` of a configured MinIO deployment from which
```

Comment:
Rather than "retrieve the diagnostic data", `mc` is running the speedtest against this MinIO deployment.

Reply:
If changed the description in the latest commit, this should now be fine